### PR TITLE
Adding k8s best practices.

### PR DESCRIPTION
**Pull request by bridgetkromhout**
Forthcoming ep - needs recording and date adjustment.

### CODE REVIEW

File: content/guest/evillalba.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Linkedin = "edvilla"
+Pronouns = ""
+GitHub = "evillgenius75"
+Thumbnail = "img/guests/evillalba.png"
+Pinterest = ""
+Instagram = ""
+YouTube = ""
++++
+
+Eddie Villeba is a an Azure Global Cloud Architect focusing on Open Source Cloud and Kubernetes. He has helped many real-world users adopt Kubernetes for their applications.
```
Comment (evillgenius75) [inline content/guest/evillalba.md]: Eddie Villalba is a Principal Software Engineer for Microsoft's Commercial Software Engineering division where he helps some of the largest enterprises in the world with cloud native application design and DevOps practices. Eddie is based in Austin, Texas where he also co-leads the Austin Kubernetes Meet-up. He is an Air Force Veteran, brand new first time dad, Kubernetes Contributor and SIG Release member.

Comment (bridgetkromhout) [reply content/guest/evillalba.md]: Awesome - updated!

File: content/guest/levenson.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Linkedin = "lachlanevenson"
+Pronouns = ""
+GitHub = "lachie83"
+Thumbnail = "img/guests/levenson.png"
+Pinterest = ""
+Instagram = ""
+YouTube = "LachlanEvenson"
++++
+
+Lachlan is a principal program manager on the Container Compute team at Azure. He has spent the last few years working with Kubernetes and enabling cloud native journeys. Lachie serves as a Cloud Native ambassador and is the Kubernetes 1.16 release lead, and has deep operational knowledge of many Cloud Native projects.
```
Comment (lachie83) [inline content/guest/levenson.md]:
```suggestion
Lachlan is a principal program manager on the Container Compute team at Azure. He has spent the last few years working with Kubernetes and enabling cloud native journeys. Lachie serves as a Cloud Native ambassador and was the Kubernetes 1.16 release lead, and has deep operational knowledge of many Cloud Native projects.
```

**Comment by bridgetkromhout** [comment]
We'll publish this episode when the book is available (which will likely be in a few weeks).

**Pull request merged**